src/odemis/driver/test/ueye_test.py

```diff
@@ -37,7 +38,7 @@
 TEST_NOHW = (os.environ.get("TEST_NOHW", "0") != "0")  # Default to Hw testing
 
 CLASS = ueye.Camera
-KWARGS = dict(name="camera", role="ccd", device=None, transp=[2, -1])
+KWARGS = dict(name="camera", role="ccd", device=None, max_res=None, transp=None)
```

Comment:
I think you could leave the transp as is. It's always good to test with `transp`, it finds extra bugs ;-)

Reply:
Please, leave `transp=[2, -1]` always there.